File: QMPlusServer/controller/api/workFlow.go
package api

import (
	"fmt"
	"github.com/gin-gonic/gin"
	"main/controller/servers"
	"main/model/dbModel"
)

// @Tags workflow
// @Summary 注册工作流
// @Produce  application/json
// @Param data body dbModel.Workflow true "注册工作流接口"
// @Success 200 {string} json "{"success":true,"data":{},"msg":"注册成功"}"
// @Router /workflow/createWorkFlow [post]
func CreateWorkFlow(c *gin.Context) {
	var wk dbModel.Workflow
	_ = c.ShouldBind(&wk)
	err := wk.Create()
	if err != nil {
		servers.ReportFormat(c, false, fmt.Sprintf("获取失败：%v", err), gin.H{})
	} else {
		servers.ReportFormat(c, true, "获取成功", gin.H{})
	}
}

File: web/src/api/workflow.js
import service from '@/utils/request'
// @Summary 删除角色
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body {authorityId uint} true "删除角色"
// @Success 200 {string} json "{"success":true,"data":{},"msg":"获取成功"}"
// @Router /authority/deleteAuthority [post]
export const createWorkFlow = (data) => {
    return service({
        url: "/workflow/createWorkFlow",
        method: 'post',
        data
    })
}